2016/ideas-list-swc.md

```diff
@@ -200,32 +221,18 @@ Skills required to complete the project (you can apply without some of them):
   for storing the results.
 * Knowledge of [Git][] for storing your work on [GitHub][]
 
-Any of these skills could be learned during the project and I will help you
+Any of these skills could be learned during the project and we will help you
```

Comment:
Again, first `Skills required […]`, then `Any of these skills […]`, and finally:

```markdown
This project will consist of single server application you'll be designing and writing from scratch, and you'll also make changes to existing scripts too.
```